crates/bevy_pbr/src/light/point_light.rs

```diff
@@ -22,29 +22,60 @@ use super::*;
 pub struct PointLight {
     /// The color of this light source.
     pub color: Color,
+
     /// Luminous power in lumens, representing the amount of light emitted by this source in all directions.
     pub intensity: f32,
+
     /// Cut-off for the light's area-of-effect. Fragments outside this range will not be affected by
     /// this light at all, so it's important to tune this together with `intensity` to prevent hard
     /// lighting cut-offs.
     pub range: f32,
+
     /// Simulates a light source coming from a spherical volume with the given radius. Only affects
     /// the size of specular highlights created by this light. Because of this, large values may not
     /// produce the intended result -- for example, light radius does not affect shadow softness or
     /// diffuse lighting.
     pub radius: f32,
+
     /// Whether this light casts shadows.
     pub shadows_enabled: bool,
+
+    /// Whether soft shadows are enabled, and if so, the size of the light.
+    ///
+    /// Soft shadows, also known as *percentage-closer soft shadows* or PCSS,
+    /// cause shadows to become blurrier (i.e. their penumbra increases in
+    /// radius) as they extend away from objects. The blurriness of the shadow
+    /// depends on the size of the light; larger lights result in larger
+    /// penumbras and therefore blurrier shadows.
+    ///
+    /// Currently, soft shadows are rather noisy if not using the temporal mode.
+    /// If you enable soft shadows, consider choosing
+    /// [`ShadowFilteringMethod::Temporal`] and enabling temporal antialiasing
+    /// (TAA) to smooth the noise out over time.
+    ///
+    /// Note that soft shadows are significantly more expensive to render than
+    /// hard shadows.
+    pub soft_shadow_size: Option<f32>,
```

Comment:
Is there a reason to use a separate field for the size of the light source here when there is already a `radius` field? I suppose if gives more not-physically-correct artistic control over the shadow softness independent of the specular highlights.

Reply:
I removed the separate field in favor of just using `radius`.